### DISCUSSION TITLE
Footers in the DOCX file are not displayed in the OpenOffice and LibreOffice

### DISCUSSION THREAD

**Discussion by Uniserg** [eclipse-birt/birt#1920] [General]
Hi! I'm using BIRT 4.13. 
Problem is:
Footers (Master Page) in the DOCX file are not displayed in the OpenOffice and LibreOffice, whereas they are displayed normally in Word. Also, if I save the DOCX in Word and then open it in OpenOffice/LibreOffice, the footers appear. It looks like the problem is related to the DOCX emitter.

Could you let me know if this issue has been fixed in the subsequent versions of BIRT? If so, please provide a link to the relevant commit.

Thank you in advance for your help!

**Comment by merks** [comment]
There's nothing stopping your from testing the 4.17 release yourself is there?

https://download.eclipse.org/birt/updates/release/4.17.0/

**Reply by speckyspooky** [reply]
The requirement to create DOCX and afterwards it should be displayed accordingly with OpenOffice
is not given in all cases so we will never create DOCX to be fine with OpenOffice.

There are different ways internally to create footers based on WordProcessingML.
